README.md

```diff
@@ -234,7 +234,7 @@ let mut table = Table::new(&data);
 table.with(Style::psql());
 ```
 
-Below is a rendered list of the preconfigured styles.
+Below is a rendered list of the incl styles.
```

Comment:
not sure `incl` is clearer

especially as a non native english speaker

Reply:
agreed